influxdb/1.3/alpine/Dockerfile

```diff
@@ -5,6 +5,7 @@ RUN echo 'hosts: files dns' >> /etc/nsswitch.conf
 ENV INFLUXDB_VERSION 1.3.0
 RUN set -ex && \
     apk add --no-cache --virtual .build-deps wget gnupg tar ca-certificates && \
+    apk add --no-cache tzdata && \
```

Comment:
Can you do this with the same pattern as `telegraf/1.3/alpine`? The `apk add` should be a separate command that happens before the `ENV` call. Can you also squash your commits into 1 commit?